create-cluster/deploy.sh

```diff
@@ -63,10 +64,17 @@ installConfig:
 $(applier -d hub/$CLOUD/install_config.yaml $IN -o /dev/stdout -s | sed 's/^/  /')
 EOF)
 
+# Create the managed cluster
 if [ -z ${DEL+x} ]
 then
-    echo "$EXT_VALUES" | applier -d hub/common $IN $OUT -s $VERBOSE
+  # vSphere needs an extra secret for CA cert
+  echo "$EXT_VALUES" | applier -d hub/common $IN $OUT -s $VERBOSE
+  if [ ${CLOUD} == "vsphere" ]
+  then
+    echo "$EXT_VALUES" | applier -d hub/vsphere/vsphere_ca_cert_secret_cr.yaml $IN ${OUT} -s $VERBOSE
```

Comment:
instead of doing that you could move that yaml in the common and add around it a {{ if }} to check if the cloud==vsphere.
I know common is supposed to be common but isn't it nicer?
so you don't have to check the type of cloud in the shell.